proto/rill/runtime/v1/queries.proto

```diff
@@ -938,6 +938,8 @@ message TimeSeriesValue {
 message TableCardinalityRequest {
   string instance_id = 1;
   string connector = 4;
+  string database = 5;
+  string db_schema = 6;
```

Comment:
It's missing handling for a bunch of the other analytical APIs – try searching for `string table_name`

Reply:
I only added support for `Tablewide profiling APIs` bcz those are the ones supposed to be used for external tables. Do we want this support in other profiling APIs as well ?

Reply:
Added support in other APIs as well for future use cases.